front/src/applications/stdcm/components/StdcmHelpModule/StdcmHelpModule.tsx

```diff
@@ -0,0 +1,84 @@
+import { useState } from 'react';
+
+import { ChevronRight, X } from '@osrd-project/ui-icons';
+import cx from 'classnames';
+import { useTranslation } from 'react-i18next';
+
+import HelpSection from './HelpSection';
+
+type StdcmHelpSectionProps = {
+  toggleHelpModule: () => void;
+  showHelpModule: boolean;
+};
+
+const StdcmHelpModule = ({ toggleHelpModule, showHelpModule }: StdcmHelpSectionProps) => {
+  const { t } = useTranslation('stdcm-help-section');
+
+  const [activeSection, setActiveSection] = useState<string | null>(null);
+
+  const closeHelpModule = () => {
+    setActiveSection(null);
+    toggleHelpModule();
+  };
+  const closeHelpSection = () => setActiveSection(null);
+  const support = t('asu', { returnObjects: true }) as Array<Record<string, string>>;
+  const sections = Object.keys(t('sections', { returnObjects: true }));
+  return (
+    <div className={cx('stdcm__help-module', { active: showHelpModule })}>
+      <div className="stdcm__help-module__header">
+        <button type="button" className="stdcm__help-module__close" onClick={closeHelpModule}>
+          <X size="lg" iconColor="#B6B2AF" />
+        </button>
+      </div>
+      <div className="stdcm__help-module__content">
+        <h1 className="stdcm__help-module__title">{t('help')}</h1>
+        <div className="stdcm__help-module__chapters">
+          {sections &&
+            sections.map((section, index) => (
+              <>
+                <button
+                  type="button"
+                  key={index}
```

Comment:
Nit: doesn't really matter since the list items are static, but `section` would be slightly better to as key use here.

Reply:
yes thanks